pkg/cloud/aws.go

```diff
@@ -0,0 +1,30 @@
+package cloud
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func GetAWSResources() OwnedResources {
+	return OwnedResources{
+		newTestResource(),
+	}
+}
+
+func newTestResource() *v1.Service {
+	svc := &v1.Service{
```

Comment:
Tbh, I don't like the idea of creating these objects manually. I would prefer to automate it and generate them from yaml files. Here is an example https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/10

Reply:
Let's shelf it for now and return once we have a reference implementation with identified things we need to configure dynamically.